packages/logging-winston/src/index.js

```diff
@@ -156,9 +156,20 @@ LoggingWinston.prototype.log = function(levelName, msg, metadata, callback) {
 
   var labels = {};
 
-  // Logging proto requires that the label values be strings, so we convert
-  // using util.inspect.
-  if (is.object(metadata)) {
+  // For error messages at severity 'error' and higher, Stackdriver
+  // Error Reporting will pick up error messages if the full stack trace is
+  // included in the textPayload or the message property of the jsonPayload.
+  // https://cloud.google.com/error-reporting/docs/formatting-error-messages
+  //
+  if (is.error(metadata)) {
+    if (msg.length === 0) {
+      msg = metadata.stack;
```

Comment:
For bunyan, `message` is a property of the log entry's data field. The data field gets formatted as a jsonPayload in the protobuf.

For winston, `msg` *was* the log entry's data field – since winston gives us a plain JS string as the message to be recorded. This gets formatted as a textPayload in the jsonPayload.

However, this all has changed now given that Error Reporting seems to prefer jsonPayload, specially if the application is running off of GCP. The latest commits always format winston log entries as a jsonPayload, so the code should be more consistent between winston and bunyan.